<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Robot1857v2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>이름</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="text">
          <string>Robot1857v2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>단위투자금</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_unit">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>매수방법</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_buy_sHogaGb">
         <item>
          <property name="text">
           <string>00:지정가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03:시장가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05:조건부지정가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06:최유리지정가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07:최우선지정가</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>매도방법</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_sell_sHogaGb">
         <item>
          <property name="text">
           <string>00:지정가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03:시장가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05:조건부지정가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06:최유리지정가</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07:최우선지정가</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>포트폴리오수</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_portsize">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_filename"/>
       </item>
       <item row="9" column="1">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>검색식</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit_buytime">
         <property name="plainText">
          <string>09:00:00-11:00:00,
12:00:00-13:00:00,
14:00:00-15:20:00</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>매수거래시간</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="lineEdit_trailstop">
         <property name="text">
          <string>0.01</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>트레일링스탑</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="lineEdit_sellall">
         <property name="text">
          <string>15:15:00</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>일괄매도시각</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>매도거래시간</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit_selltime">
         <property name="plainText">
          <string>09:00:00-11:00:00,
12:00:00-13:00:00,
14:00:00-15:20:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>SearchFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gen_range()</slot>
  <slot>SearchFile()</slot>
 </slots>
</ui>
